templates/Segment_Image_Data.ipynb

```diff
@@ -344,13 +344,13 @@
    ],
```

Comment:
<p>We should just change xr_dim_name from compartments to channels. Not sure why it was set to compartments here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>